### ISSUE TITLE
[MU4 Issue] Muse Sounds: Sound artifact when opening score or adding articulation (stacc, accent)

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#13826]
**To Reproduce**
1. Muse Sounds should be installed
2. Open score (or open [Smoke Test 17.zip](https://github.com/musescore/MuseScore/files/9814200/Smoke.Test.17.zip) )
3. Open mixer while loading sounds  - **sound artifact appears for a moment (Mixer shows that signal)**
4. Select some notes
5. Add staccato - **sound artifact appears for a moment**

https://user-images.githubusercontent.com/90187801/196519589-8cc8a952-51a1-4625-a04f-d91f0d9757d5.mp4

**Platform information**
 macOS, Windows, Linux

**Comment by DmitryArefiev** [comment]
Could be related to #13532

**Comment by cbjeukendrup** [comment]
I think that #13603 is the same problem as this. So that's _not_ just me, and _not_ just macOS. Interesting...

**Comment by DmitryArefiev** [comment]
> Also, good that you found that it happens only when Muse Sounds is installed.

Yeah, the issue occurs with Muse Sounds installed and activated in score

**Comment by cbjeukendrup** [comment]
Another related issue that has very probably the same root cause: #14695

**Comment by abariska** [comment]
Also, one metronome click is played right after a score created when metronome is enabled

**Comment by mccoydtromb** [comment]
This is still happening for me on latest nightly and with all updates.

**Comment by Tantacrul** [comment]
Yeah, I'm still getting this too, especially on larger scores. @davidwhearn - the StarWars score is a good one to test.

However, I'm marking this as a P2, since I think P1 is perhaps a bit high.

**Comment by Valdessir** [comment]
I would like to add that there's a sound artifact as well when typing staff text, or editing them via Properties.

**Comment by Tantacrul** [comment]
> I would like to add that there's a sound artifact as well when typing staff text, or editing them via Properties.

Yes I've noticed this too
